### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Markolie**
Fixes https://github.com/ParadiseSS13/Paradise/issues/5747.
Fixes https://github.com/ParadiseSS13/Paradise/issues/5761.
Fixes https://github.com/ParadiseSS13/Paradise/issues/5726.

🆑 Markolie
bugfix: Cryo tubes no longer expel air.
bugfix: Mechas will once again take damage from projectiles.
bugfix: Pods now properly trigger projectiles. This means they can now be damaged by ion weaponry properly.
/🆑

### CODE REVIEW

**Comment by Iamgoofball** [comment]
Did you ignore the comment

**Comment by Markolie** [comment]
Nope, I did not. With expel_gas disabled, characters do not take damage while inside or out of the cryotube, nor do they suddenly die as the comment describes.

**Comment by LittleBigKid2000** [comment]
Did you test it with Unathi and Slime People?

**Comment by Markolie** [comment]
I just double checked, no damage there. Both /tg/ and Bay removed this proc a long time ago, by the way. It's probably a relic.

File: code/modules/spacepods/spacepod.dm [hunk 1]
```diff
@@ -284,35 +283,27 @@
 /obj/spacepod/emp_act(severity)
 	occupant_sanity_check()
 	cargo_hold.emp_act(severity)
+	
+	if(battery && battery.charge > 0)
+		battery.use((battery.charge / 2) / severity)
+	deal_damage(80 / severity)
```
Comment (Markolie) [inline code/modules/spacepods/spacepod.dm]: I've made some minor changes to the charge reduction/damage dealt by ion weapons here. The cell charge reduction now scales so two ion shots don't drain the entire pod, and the pod can now survive one more ion shot (a full ion gun will still kill it). These changes are based off of mech's.

I doubt the ion damage on pods was every properly tested (as it's been broken for a while now), which is why these numbers had to be tweaked a bit after testing.

File: code/game/mecha/mecha.dm [hunk 2]
```diff
@@ -563,7 +563,9 @@
 			dam_coeff = B.damage_coeff
 			break
 
-	if(prob(deflect_chance & deflection) && (Proj.damage_type == BRUTE || Proj.damage_type == BURN))
+	if(prob(deflect_chance * deflection) && (Proj.damage_type == BRUTE || Proj.damage_type == BURN))
+		visible_message("<span class='danger'>[src]'s armour deflects [Proj]!</span>")
+	else
 		visible_message("<span class='danger'>[src] is hit by [Proj].</span>")
 		take_damage(Proj.damage * dam_coeff, Proj.flag)
 		check_for_internal_damage(list(MECHA_INT_FIRE,MECHA_INT_TEMP_CONTROL,MECHA_INT_TANK_BREACH,MECHA_INT_CONTROL_LOST,MECHA_INT_SHORT_CIRCUIT))
```
Comment (Fox-McCloud) [inline code/game/mecha/mecha.dm]: Yeah, I realized that, thus why I deleted my comment---too late, you commented on it anyway. Sad times.

Comment (Markolie) [inline code/game/mecha/mecha.dm]: Such is life!

Comment (Markolie) [inline code/game/mecha/mecha.dm]: Deflection is always 1, and is modified by exosuit equipment (the armor booster raises it to 1.15). The base armor of a mech (usually around 30) is multiplied by the deflection and that in total determines the deflection chance. Having only deflection in there would render the armor booster useless.

**Comment by GelatinousGlob** [comment]
I think it is cool that cryotubes spew oxygen its just for some reason it was spewing way too much way too fast.

I think it would be preferable if it still spewed oxygen but just less of it. And right now the effect isn't even apparent, but if the gas that escape was say slightly lower temperature than the room instead of exactly room temperature it would give a slight chill around it but not enough to harm someone.

**Comment by Markolie** [comment]
It's a pointless feature implemented to shoddily fix a but that's no longer around and causes more issues than it's worth. I'd rather not keep it around.

**Comment by Plotron** [comment]
That 'Mech/Pod bug is quite serious. Why can't this be merged faster?

**Comment by Fox-McCloud** [comment]
@Plotron 

Need to run the cryo bit by @TheDZD otherwise it's ready.

**Pull request merged**